source/whats-new.txt

```diff
@@ -28,8 +28,28 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Ability to define your own custom type for the top level document
+  returned in a change event.
+- The ``ChangeStreamDocument`` type now defaults to union all the
+  possible events.
+- Added `the following properties
+  <{+api+}/modules.html#ChangeStreamDocument>`__ : ``to``, 
```

Comment:
Issue: link text doesn't describe the page you're linking to, see the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/)

Suggestion: "Added the following properties to ChangeStreamDocument:" (link ChangeStreamDocument)